blocks/url-input/button.js

```diff
@@ -38,12 +38,13 @@ class UrlInputButton extends Component {
 	render() {
 		const { url, onChange } = this.props;
 		const { expanded } = this.state;
+		const buttonLabel = url ? "Edit Link" : "Insert Link";
```

Comment:
ESLint is causing a build failure here since these should be single quote strings:

>WordPress uses single quotation marks for string declarations.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/